node-backend/lib/silent-learner/memory-pack.mjs

```
@@ -0,0 +1,307 @@
+    task: event.task_type || 'general',
+    source: event.source,
+    outcome: event.outcome,
+    files: event.files_touched || [],
+    timestamp: event.created_at,
+  };
+
+  // Add pack-specific fields
+  switch (packType) {
+    case 'accepted-solutions':
+      lesson.signal = 'accepted';
+      let metaChangeCount = (event.metadata?.fileChangeCount || 0) + (event.metadata?.artifactChangeCount || 0);
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)